tests/unit/rules/javascript/test.incompatible_browser_api.js

```diff
@@ -37,7 +37,7 @@ describe('incompatible browser APIs', () => {
   });
 
   it('does not flag APIs that are not implemented on Android', async () => {
-    const code = 'browser.browserAction.setBadgeText({ text: "lorem ipsum" });';
```

Comment:
@wagnerand thanks for looking into it, from the change I see that the test was expected to fail because the API is now implemented in Fenix.

Maybe we should replace that API call with one that we are not going to implement for Fenix, the `bookmarks` API seems something that we may actually implement at some point, would you mind to try to see if we can replace that call with a call to a `sidebarAction` API method? (the sidebarAction API seems one that is more unlikely to be implemented for Fenix)

Reply:
577b814b